packages/application-shell/src/components/quick-access/quick-access.spec.js

```diff
@@ -1045,4 +1258,102 @@ describe('QuickAccess', () => {
       expect(getByText('Open PIM Search')).toBeVisible();
     });
   });
+
+  describe('when searching for a product with enabled pim-search', () => {
+    it('should find a product by full-text pim search', async () => {
+      const searchText = 'party';
+      const partyParrotId = '7a66d631-0e52-44f8-bb5d-2212839a2519';
+      const mocks = [
+        {
+          request: {
+            query: QuickAccessQuery,
+            variables: {
+              searchText,
+              canViewProducts: true,
+              target: 'ctp',
+              productsWhereClause: `id in ("${partyParrotId}")`,
+              includeProductsByIds: true,
+            },
+          },
+          result: {
+            data: {
+              productsByIds: {
+                results: [
+                  {
+                    id: partyParrotId,
+                    masterData: {
+                      staged: {
+                        nameAllLocales: [
+                          { locale: 'en', value: 'Party Parrot' },
+                        ],
+                      },
+                    },
+                  },
+                ],
+              },
+              productById: null,
+              productByKey: null,
+              productByVariantSku: null,
+              productByVariantKey: null,
+            },
+          },
+        },
+      ];
+      const { getByTestId, getByText } = renderWithRedux(
+        <QuickAccess {...createTestProps()} />,
+        {
+          mocks,
+          sdkMocks: [
```

Comment:
This is the best example of how the new `sdkMocks` option in `renderWithRedux` can be used to mock sdk responses.